### ISSUE TITLE
Stops working if vim is focused

### ISSUE THREAD

**Issue by RidaAyed** [jwilm/i3-vim-focus#5]
Hi

As soon as vim/gvim gets focus, the function stops working. Any idea what else I could try?

My .vimrc
```
""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
"                                  Plugins                                   "
""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
set rtp+=~/.vim/bundle/Vundle.vim " set the runtime path to include Vundle and initialize
call vundle#begin()
Plugin 'VundleVim/Vundle.vim'     " let Vundle manage Vundle, required
                                  " Keep Plugin commands between vundle#begin/end.
                                  " plugin on GitHub repo
Plugin 'jwilm/i3-vim-focus'
" Put your non-Plugin stuff after this line
call vundle#end()            " required
filetype plugin indent on    " required
""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
"                               PluginSettings                               "
""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
"" i3/vim navigation
map gwl :call Focus('right', 'l')<CR>
map gwh :call Focus('left', 'h')<CR>
map gwk :call Focus('up', 'k')<CR>
map gwj :call Focus('down', 'j')<CR>
```

**Comment by blacksails** [comment]
I have the same issue.

**Comment by jwilm** [comment]
Also not working for me recently but I haven't made time to fix it :(

**Comment by XyloVech** [comment]
In your .i3/config file add:

`bindsym --release $mod+Left exec "i3-vim-focus left"`
`bindsym --release $mod+Down exec "i3-vim-focus down"`
`bindsym --release $mod+Up exec "i3-vim-focus up"`
`bindsym --release $mod+Right exec "i3-vim-focus right"`


instead of the recommended bindings from the project's readme.
The `--release` does the trick by triggering the binding upon key release.

**Comment by jwilm** [comment]
Resolved by #6
